deploy/storage/base_bigtable_info.yaml

```diff
@@ -4,7 +4,7 @@ tables:
   - auto2w_2023_07_25_10_35_37
   - auto1d_2023_07_31_21_08_49
   - auto1w_2023_07_30_21_04_03
-  - frequent_2023_07_31_20_05_22
+  - frequent_2023_08_10_20_03_02
```

Comment:
Can this be bumped to the latest: frequent_2023_08_18_02_22_08 to get the recent worldbank import?

Reply:
done